utils/xmllite.d

```diff
@@ -44,6 +44,8 @@ private struct StringStream
 
 	char read() { return s[position++]; }
 	@property size_t size() { return s.length; }
+	size_t backup() const { return position; }
```

Comment:
`position` is accessed everywhere, so there is no need for these helper functions.

Reply:
The idea is that the calling code should not need to worry about what it takes to backup and restore the StringStream.